.eleventy.js

```diff
@@ -20,18 +20,36 @@ module.exports = function(config) {
-   config.addPassthroughCopy({"src/lib": "lib"});
-   config.addPassthroughCopy({"src/assets/stylesheets": "assets/stylesheets"});
-   config.addPassthroughCopy({"src/posts/images": "posts/images"})
+  // Passthrough copy
+  config.addPassthroughCopy({"src/assets/images": "assets/images"});
+  config.addPassthroughCopy({"src/assets/js": "assets/js"});
+  config.addPassthroughCopy({"src/assets/stylesheets": "assets/stylesheets"});
+  config.addPassthroughCopy({"src/posts/images": "posts/images"})
+  config.addPassthroughCopy({"node_modules/infusion/dist/infusion-all.min.js": "lib/infusion/infusion-uio.min.js"});
+  config.addPassthroughCopy({"node_modules/infusion/src/components/tableOfContents/css/": "lib/infusion/src/components/tableOfContents/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/components/tableOfContents/html/": "lib/infusion/src/components/tableOfContents/html/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/components/textfieldControl/css/": "lib/infusion/src/components/textfieldControl/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/components/switch/css/": "lib/infusion/src/components/switch/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/Infusion-LICENSE.txt": "lib/infusion/Infusion-LICENSE.txt"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/preferences/css/": "lib/infusion/src/framework/preferences/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/preferences/fonts/": "lib/infusion/src/framework/preferences/fonts/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/preferences/html/": "lib/infusion/src/framework/preferences/html/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/preferences/messages/": "lib/infusion/src/framework/preferences/messages/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/preferences/preferencesDependencies.json": "lib/infusion/src/framework/preferences/preferencesDependencies.json"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/enhancement/css/": "lib/infusion/src/framework/enhancement/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/core/css/": "lib/infusion/src/framework/core/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/framework/core/images/": "lib/infusion/src/framework/core/images/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/lib/jquery/ui/css/default-theme/": "lib/infusion/src/lib/jquery/ui/css/default-theme/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/lib/normalize/css/": "lib/infusion/src/lib/normalize/css/"});
+  config.addPassthroughCopy({"node_modules/infusion/src/lib/open-dyslexic/fonts/": "lib/infusion/src/lib/open-dyslexic/fonts"});
+  config.addPassthroughCopy({"node_modules/infusion/src/lib/opensans/fonts/": "lib/infusion/src/lib/opensans/fonts/"});
```

Comment:
Some/all of these can actually be sourced from `dist/assets/` as opposed to `src`. However, if things are working this way, I suppose you can leave it until after [FLUID-6359](https://issues.fluidproject.org/browse/FLUID-6359) is addressed.

Reply:
Leaving this for now, will revisit when [FLUID-6359](https://issues.fluidproject.org/browse/FLUID-6359) lands.